app/lib/methods/callJitsi.ts

```diff
@@ -30,21 +32,21 @@ async function jitsiURL({ room }) {
 	if (Jitsi_URL_Room_Hash) {
 		rname = uniqueID + room?.rid;
 	} else {
-		rname = encodeURIComponent(room.t === 'd' ? room?.usernames?.join?.(' x ') : room?.name);
+		rname = encodeURIComponent(room.t === 'd' ? room?.usernames?.join?.(' x ') || '' : room?.name);
```

Comment:
Can we change this `d` to `SubscriptionType.DIRECT` ?

Reply:
done